Cython/Compiler/Code.py

```diff
@@ -237,6 +237,13 @@ class UtilityCodeBase(object):
 
         [definitions]
 
+        ##### MyUtility #####
+        #@subsitute: tempita
+
+        [requires tempita substitution
+         - context can't be specified here though
```

Comment:
Then, where does it get the substitution context/variables from?

Reply:
It doesn't - this is a limitation. It only works on things that don't need a context. For example the two that I've used it for `GetItemInt` and `SliceTupleAndList` just do a loop over a short list of names and don't need any extra context data. It's obviously limited, but the simple case solved the problem I was having.

Maybe the comment should be changed to explain that a little better?